en/simulation/README.md

```diff
@@ -140,3 +140,32 @@ The simulated camera is a gazebo plugin that implements the [MAVLink Camera Prot
 
 The same approach can be used by other simulators to implement camera support.
 
+## Running Simulation on the Remote Server 
+
+As had been mentioned at the beginning the simulation environment can be run on multiple computers on the same network. Unfortunately, it is a slightly complicated, because of the out of box configuration does not broadcast the PX4 UDP packets to external interfaces and packets are routed internally by default. 
```

Comment:
So just to spell it out for me, the internal default routing just means that we route to localhost?

> If you're on the same network, the stream can also be sent to a certain IP directly (via UDP), instead of broadcasting.

Is this the -t option in http://dev.px4.io/en/middleware/modules_communication.html#mavlink_usage ?
WOuld I be correct in assuming that this means that to route to an external port you would need to change the calls in code that launch mavlink and talk to 14550 (say) to use the -t and an IP address?

> If you're not in the same network, a VPN is also an option.

Is there any setup needed to discuss for this? Or is it as simple as setting your system up to use VPN for external traffic and updating to use external network address as per previous point above?

Sorry if these are dumb questions - I just don't want to write "use VPN" if that then means someone has to do a whole lot of working out of stuff.

Reply:
> So just to spell it out for me, the internal default routing just means that we route to localhost?

Right.

> Is this the -t option in http://dev.px4.io/en/middleware/modules_communication.html#mavlink_usage ?
WOuld I be correct in assuming that this means that to route to an external port you would need to change the calls in code that launch mavlink and talk to 14550 (say) to use the -t and an IP address?

Correct.

> Is there any setup needed to discuss for this? Or is it as simple as setting your system up to use VPN for external traffic and updating to use external network address as per previous point above?

Nothing specifically. If you have the VPN, you're essentially back to the local network case (assuming the VPN routes all ports, which is usually the default).

> Sorry if these are dumb questions - I just don't want to write "use VPN" if that then means someone has to do a whole lot of working out of stuff.

Not dumb questions! VPN is a large topic in itself and there are many variations, and therefore lots of things that can go wrong. But I would not go into that here, I think someone setting up a VPN for that needs to have some networking background anyway. What's important is to have good documentation which ports we use.